lib/features.js

```diff
@@ -63,6 +63,12 @@ const features = {
         packages: ['vue', 'vue-loader', 'vue-template-compiler'],
         description: 'load VUE files'
     },
+    eslint: {
+        method: 'enableEslintLoader()',
+        // eslint is needed so the end-user can do things
+        packages: ['eslint', 'eslint-loader', 'babel-eslint', 'eslint-plugin-import'],
```

Comment:
I don't quite understand what the `eslint-plugin-import` is supposed to do - obviously something related to parsing imports, etc :). Would it be possible to enhance the `fixtures/eslint.js` file to take advantage of a feature supplied by this library, so we can make sure it's configured correctly?

Reply:
given this is only useful if you actually use es2015+ import syntax, maybe even just remove it, no?

Reply:
If we remove it, and someone *does* use `import`, what happens? What do we lose? Do we just lose some feature of eslint that validates that those paths are correct? Or is it something different?

Reply:
Just re-read the docs. if you don't use the import sytnac, but e.g. a commonJS or AMD syntax, then this will throw errors about this.
Personally, I prefer going for "enable by default" since this is my personal (and companies) use-case. However i'm not sure if we can simply enforce it. It should be easily disabled/enabled atleast.

Maybe look into having a way of advising users to install a package? or checking at run-time if it's installed or not?

Reply:
Hmm. Let me see if I understand this correctly:

A) *Without* `eslint-plugin-import`, any `import` statements aren't validated, so in theory, you could be importing a non-existent path and would go no lint error. Are normal `require` paths linted?

B) *With( `eslint-plugin-import`, `import` statements *are* validated, but now you MUST use `import` in all situations: any calls to `require` will fail.

Is this correct? Actually, when I'm reading, I think (B) might be *incorrect*. I see the rule you're talking about - `eslint-plugin-import`. But under [Installation](https://www.npmjs.com/package/eslint-plugin-import#installation), it talks about how all the rules are disabled by default.

If we're not sure, then "less" and more "default" setup is better. The most important thing is that we're creating a really easy "system" for enabling eslint AND for adding your own plugins.

Reply:
See my comment in the other track, I think it's best to remove this required dependency.

FYI, require still works. This plugin just analyses your import statements.

Reply:
I’ll take a closer look at your updates and comments soon, but yea, let’s remove this :).

Thanks for your work!

Reply:
@weaverryan Anything left for me on this MR?